src/report.js

```diff
@@ -44,6 +45,10 @@ class Report {
       memoryLimitInMB
     } = this.context;
 
+    const pluginConfigss = plugins.map(pluginFn => {
```

Comment:
Agree. I'm proposing `name` and `version` -- with `version` maybe being optional?